packages/spec/helpers/env.js

```diff
@@ -113,7 +113,9 @@ class FixtureEnvironment extends NodeEnvironment {
 
   async teardown() {
     await super.teardown();
-    await this.closeBrowser();
+    if (this.closeBrowser) {
+      await this.closeBrowser();
```

Comment:
Don't we have optional chaining now? Couldn't you use `this?.closeBrowser()`?
But that may depend on the node version we have here.

Reply:
I think it requires at least Node.js v14. This change isn't really related to this PR but it hides other errors because it fails if the process could start the browser in the first place.